### PR TITLE
Fixed the account helpers to comply with the multi-store params

### PR DESCRIPTION
**Pull request by mridang**

### CODE REVIEW

File: classes/models/customer.php [hunk 1]
```diff
@@ -51,12 +51,11 @@ class NostoTaggingCustomer extends NostoTaggingModel
     /**
      * Loads the customer data from supplied context and customer objects.
      *
-     * @param Context $context the context object.
      * @param Customer $customer the customer object.
      */
-    public function loadData(Context $context, Customer $customer)
+    public function loadData(Customer $customer)
```
Comment (oostman) [inline classes/models/customer.php]: Did not figure out where this is called from, but did you change caller as well?

File: classes/helpers/account.php [hunk 2]
```diff
@@ -171,14 +205,14 @@ public function existsAndIsConnected($lang_id = null, $id_shop_group = null, $id
...
-    public function updateCurrencyExchangeRates(NostoAccount $account, Context $context)
+    public static function updateCurrencyExchangeRates(NostoAccount $account, Context $context)
     {
         /** @var NostoTaggingHelperCurrency $currency_helper */
         $currency_helper = Nosto::helper('nosto_tagging/currency');
         try {
             $exchangeRates = $currency_helper->getExchangeRateCollection($context);
             $service = new NostoOperationExchangeRate($account, $exchangeRates);
-            return $service->update($exchangeRates);
+            return $service->update();
```
Comment (mridang) [inline classes/helpers/account.php]: Yes. That method doesn't take parameters

File: classes/helpers/config.php [hunk 3]
```diff
@@ -424,7 +426,7 @@ public function useMultipleCurrencies($id_lang, $id_shop_group = null, $id_shop
     /**
      * Clears tagging related caches (compiled templates)
      *
-     * @param Smarty $smarty
+     * @param $smarty
```
Comment (mridang) [inline classes/helpers/config.php]: The underlying function isn't type hinted so passing Smarty to it, confuses it.

File: classes/models/customer.php (hunk 1 above)
Comment (mridang) [inline classes/models/customer.php]: Yes

File: classes/helpers/account.php [hunk 4]
```diff
@@ -150,16 +165,35 @@ public function find($lang_id = null, $id_shop_group = null, $id_shop = null)
     }
 
     /**
+     * Checks if Nosto is installed to a given store and language
+     *
+     * @param Context $context
+     * @return bool
+     */
+    public static function isContextConnected(Context $context)
```
Comment (mridang) [inline classes/helpers/account.php]: This is used a whole lot of places.

**Pull request merged**